### ISSUE TITLE
none-ls sources missing

### ISSUE THREAD

**Issue by alexpetrean80** [nix-community/nixvim#1451]
At least `flake8` and `luacheck` are missing from the sources, sometimes in the past they were there. What happened and will they be back?

**Comment by GaetanLepage** [comment]
We have an automatic way of ensuring that the none-ls sources in nixvim are exactly the same as upstream.
If a source disappears, it is because it has been removed upstream. (Otherwise, please file a bug).

**Comment by alexpetrean80** [comment]
Yes, that's why I closed the issue. Had a lapse in judgement and forgot to check upstream, the none-ls team have good reasons to sunset those and propose replacements. Sorry
